### PR TITLE
Fix LinkType default arg hack due to module loading errors

### PR DESCRIPTION
**Pull request by dvoytenko**
@swissspidy some weird module loading error with tests. PTAL.

### CODE REVIEW

**Review by swissspidy — APPROVED** [review]

**Comment by swissspidy** [comment]
Travis is actually green: https://travis-ci.com/github/google/web-stories-wp

File: assets/src/edit-story/components/panels/link.js [hunk 1]
```diff
@@ -41,17 +41,16 @@ import { isValidUrl, toAbsoluteUrl, withProtocol } from '../../utils/url';
...
 
-const DEFAULT_LINK = createLink({ url: null, icon: null, desc: null });
-
 function LinkPanel({ selectedElements, pushUpdateForObject }) {
   const selectedElement = selectedElements[0];
   const { isFill } = selectedElement;
   const inferredLinkType = useMemo(() => inferLinkType(selectedElement), [
     selectedElement,
   ]);
+  const DEFAULT_LINK = createLink({ url: null, icon: null, desc: null });
```
Comment (dvoytenko) [inline assets/src/edit-story/components/panels/link.js:50]: Why is this necessary?
